book/getting-started/run-a-wallaroo-application-docker.md

```diff
@@ -0,0 +1,201 @@
+# Run a Wallaroo Application in Docker
+
+In this section, we're going to run an example Wallaroo application in Docker. By the time you are finished, you'll have validated that your Docker environment is set up and working correctly. If you haven't already completed the Docker setup [instructions](book/getting-started/docker-setup.md), please do so before continuing.
```

Comment:
"we're going to" -> "we will"

Reply:
"you'll" -> "you will"

Reply:
"By the time you are finished," -> "By the time we have finished,"

Note that the active voice here is "we", as in, us the authors PLUS you the reader. So the learning is done by "you", but the actions are all done by "we"